modules/CmpdReg/src/server/routes/CmpdRegRoutes.coffee

```diff
@@ -856,8 +857,31 @@ exports.updateParent = (req, resp) ->
 			resp.statusCode = 500
 			resp.end "Error trying to update parent: " + error;
 	)
-	
-	
+
+exports.swapParentStructures = (req, resp) ->
+	console.log "exports.swapParentStructures"
+
+	# TODO: Do we need to keep track of who swapped the parent structure.
+	cmpdRegCall = config.all.client.service.cmpdReg.persistence.fullpath + '/parents/swapParentStructures'
+	request(
+		method: 'POST'
+		url: cmpdRegCall
+		body: req.body
+		json: true
+		timeout: 6000000
+	, (error, response, data) =>
+		if error
```

Comment:
Just to keep things consistent, I would do `if !error` first followed by an `else`

Reply:
Sure, done.